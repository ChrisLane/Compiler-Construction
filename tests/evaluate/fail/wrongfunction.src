// This is to test that we cannot call functions that don't exist. This will not evaluate.
realfunction() {}

main() {
    wrongfunction();
}
