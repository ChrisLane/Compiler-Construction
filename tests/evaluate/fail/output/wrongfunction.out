Fatal error: exception Failure("No such function.")
